src/main/java/appeng/api/storage/ICellContainer.java

```diff
@@ -19,11 +19,4 @@
  * Represents an {@link appeng.api.networking.IGridHost} that contributes to storage, such as a ME Chest, or ME Drive.
  */
 public interface ICellContainer extends IActionHost, ICellProvider, ISaveProvider {
-
-    /**
-     * tell the Cell container that this slot should blink, the slot number is relative to the
-     *
-     * @param slot slot index
-     */
-    void blinkCell(int slot);
```

Comment:
hmm, now that I think about this, probably best to make this a default that does nothing as to keep public API compatability